2/test/run

```diff
@@ -216,10 +222,19 @@ cleanup
 sleep 5
 echo "Normal test: 32 bit"
 CIDFILE_DIR=$(mktemp --suffix=jenkins_test_cidfiles -d)
-VOLUME=`mktemp -d`
+VOLUME=$(mktemp -d)
 chmod a+rwx $VOLUME
 CONTAINER_ARGS="-u 10000:0 -e OPENSHIFT_JENKINS_JVM_ARCH=i386 "
 run_tests jenkins_test
+cleanup
+sleep 5
+echo "Test: install plugins on start"
+CIDFILE_DIR=$(mktemp --suffix=jenkins_test_cidfiles -d)
+VOLUME=$(mktemp -d)
+chmod a+rwx $VOLUME
+VALIDATE_PLUGIN=ghprb
+CONTAINER_ARGS="-u 10000:0 -e OPENSHIFT_JENKINS_JVM_ARCH=x86_64 -e INSTALL_PLUGINS=ghprb:1.35.0"
+run_tests jenkins_test
```

Comment:
lest we start installing this plugin by default, it would be good for the test to first confirm the plugin is not normally there.